input/pages/using-cql.md

```diff
@@ -22,7 +22,7 @@ For example:
 library EXM146 version '4.0.0'
 ```
 
-Snippet 4-1: Library line from [EXM146.cql](Library-EXM146-FHIR.html#cql-content)
+Snippet 4-1: Library line from [EXM146.cql](Library-EXM146-FHIR.html) cql-content
```

Comment:
I don't think this change is right, and even if it was, it's not related to updating the dependency? Can we back out all these changes to the cql-content links from this PR?